pages/linux/setfacl.md

```diff
@@ -0,0 +1,23 @@
+# setfacl
+
+> Set file access control lists.
+
+- Modify ACL of file for user with read and write access:
+
+`setfacl -m u:{{username}}:rw {{file}}`
+
+- Modify default ACL of a file for all users:
+
+`setfacl -d -m u::rw {{file}}`
+
+- Remove ACL of a file for an user:
+
+`setfacl -x u:{{username}} {{file}}`
+
+- Remove all ACL entries of a file:
+
+`setfacl -b {{file}}`
+
+- Read ACL of a file:
```

Comment:
I don't think we include another command in one command's page. You can remove this.